### DISCUSSION TITLE
[Azure Plan] Dodatkowa subskrypcja

### DISCUSSION THREAD

**Discussion by jmadejek** [AzureCommunityPL/group#363] [Pomoc] [answered]
Witam
Jak dodać dodatkową (pod)subskrypcję w Azure Plan z poziomu portalu? Robił to ktoś samodzielnie? Da się w ogóle? Szukam w dokumentacji ale nie mogę sobie poradzić...

**Comment by eklime** [comment]
A czy Azure Plan to nie jest przypadkiem zwykle Pay-as-you-go?
Bo jeśli tak, po prostu kupujesz kolejną subskrypcję i już.

**Reply by jmadejek** [reply]
Nie

**Reply by eklime** [reply]
Ach dobra, Azure Plan to ta "nowa wersja" CSP. https://docs.microsoft.com/en-us/partner-center/azure-plan-lp
No to jak jesteś klientem końcowym to raczej niewiele zdziałasz - musisz poprosić Partnera żeby Ci dorzucił kolejną subskrypcję do Twojego planu.

**Comment by mgajownik** [comment]
> 
> 
> Witam Jak dodać dodatkową (pod)subskrypcję w Azure Plan z poziomu portalu? Robił to ktoś samodzielnie? Da się w ogóle? Szukam w dokumentacji ale nie mogę sobie poradzić...

jak umowe masz tu jest step by step https://docs.microsoft.com/en-us/azure/cost-management-billing/manage/create-subscription

**Reply by jmadejek** [reply]
> jak umowe masz tu jest step by step https://docs.microsoft.com/en-us/azure/cost-management-billing/manage/create-subscription

Hej, dzięi za odpowiedź. Dotarłem do tego dokumentu. Nie za bardzo rozumiem ten zapis:

>Microsoft Partner Agreement (MPA)	Global Admin **and Admin Agent role in the CSP partner organization**. To learn more, see Partner >Center - Assign users roles and permissions. The user needs to sign to partner tenant to create Azure subscriptions.

Te zapisy tyczą się klienta końcowego? Bo ZTCW to końcowy nie ma dostępu do Partner Center.

**Comment by mgrabarz** [comment, marked as answer]
Skoro to Azure Plan w CSP, to Wasz dystrybutor lub reseller powinien dodać nowy sub w ramach planu, a nie Wy samodzielnie.

Moja firma jako reseller ma możliwość wyklikania kolejnej sub dla AP klienta w portalu dostarczonym przez dystrybutora - w tym przypadku dystrybutorem jest Also. Co ciekawe, w związku z faktem że AP wszedł w tym roku, dystrybutorzy "jeszcze" tego mogą nie wspierać w GUI, a nawet przez swoje tickety. Also pod tym względem ogarnął się szybko i wzorowo.
